lib/blueprinter/helpers/base_helpers.rb

```diff
@@ -31,6 +49,28 @@ def prepare_data(object, view_name, local_options)
         end
       end
 
+      def object_relation_to_hash(local_options, object, view_name)
+        field_to_association_hash = get_field_to_association_hash(object)
+        list_of_associations_to_preload = []
+        view_collection.fields_for(view_name).each do |field|
+          if preload_field?(field, field_to_association_hash, list_of_associations_to_preload, object)
+            list_of_associations_to_preload << field_to_association_hash[field.name]
+          end
+        end
+        object_relation_with_associations_loaded = object.preload(list_of_associations_to_preload)
```

Comment:
Does this preload nested associations?
For example, `User.preload(posts: :comments)`?

Reply:
No, it does not